packages/react/src/components/Switch/Switch.js

```diff
@@ -61,6 +61,8 @@ const Switch = React.forwardRef(function Switch(props, tabRef) {
   );
 });
 
+Switch.displayName = 'Switch';
```

Comment:
@joshblack are we doubling down on the display names? I thought we wanted to keep the `forwardRef(Switch)` for the wrapper component.

Reply:
I only suggested this since the PR is adding in `forwardRef`, so to keep things consistent with the old component the `displayName` felt appropriate.